templates/taxi/driver_list.html

```diff
@@ -9,6 +9,9 @@
           <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
```

Comment:
The URL `{% url "taxi:driver-detail" pk=driver.id %}` is correctly used to link to the driver's detail page. Ensure that the URL pattern `driver-detail` is defined in your `taxi` app's URLs.